src/main/java/com/feywild/feywild/block/FeyMushroomBlock.java

```diff
@@ -0,0 +1,88 @@
+package com.feywild.feywild.block;
+
+import io.github.noeppi_noeppi.libx.base.BlockBase;
+import io.github.noeppi_noeppi.libx.mod.ModX;
+import net.minecraft.client.renderer.ItemBlockRenderTypes;
+import net.minecraft.client.renderer.RenderType;
+import net.minecraft.core.BlockPos;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.world.level.BlockGetter;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.Blocks;
+import net.minecraft.world.level.block.BonemealableBlock;
+import net.minecraft.world.level.block.state.BlockBehaviour;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.StateDefinition;
+import net.minecraft.world.level.block.state.properties.IntegerProperty;
+import net.minecraft.world.phys.shapes.CollisionContext;
+import net.minecraft.world.phys.shapes.VoxelShape;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.api.distmarker.OnlyIn;
+
+import javax.annotation.Nonnull;
+import java.util.Random;
+import java.util.function.Consumer;
+
+public class FeyMushroomBlock extends BlockBase implements BonemealableBlock {
+
+    protected static final VoxelShape SHAPE = Block.box(5.0D, 0.0D, 5.0D, 11.0D, 6.0D, 11.0D);
+    protected static final IntegerProperty LIGHT_LEVEL = IntegerProperty.create("light_level_mushroom", 0, 15);
+
+    public FeyMushroomBlock(ModX mod) {
+        super(mod, BlockBehaviour.Properties.copy(Blocks.RED_MUSHROOM)
+                .lightLevel((mushroom) -> mushroom.getValue(LIGHT_LEVEL)));
+
+        this.registerDefaultState(this.stateDefinition.any()
+                .setValue(LIGHT_LEVEL, 7));
+    }
+
+    /*
+    @Override
```

Comment:
Why is that commented out? Please either add another comment, why this is here or remove it.